libs/block-editor/src/lib/extensions/components/suggestions/suggestions.component.html

```diff
@@ -2,17 +2,19 @@
     <h3>{{ title }}</h3>
 
     <dotcms-suggestion-list class="suggestion-list-container" #list>
-        <dotcms-suggestions-list-item
-            *ngFor="let item of items; let i = index"
-            (mousedown)="onMouseDown($event, item)"
-            (mouseenter)="onMouseEnter($event)"
-            [command]="item.command"
-            [index]="i"
-            [label]="item.label"
-            [url]="item.icon"
-            [data]="item.data"
-        >
-        </dotcms-suggestions-list-item>
+        <ng-container *ngFor="let item of items; let i = index">
```

Comment:
Why?

Reply:
not needed anymore, but, two structural directives are not supported in the same element.